AGENTS.md

```
@@ -296,15 +297,15 @@ KB          阶段 0–N 只读（决策 D3）：sliceKB 返回空对象/极简
 
 ## 7. 工作流：EPCC-V（每个任务必须遵循）
 
-新会话一律先执行 `/init-session-agora`。
+新会话一律先执行 `$agora-init-session`。
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)